app/src/main/java/com/duckduckgo/app/statistics/VariantManager.kt

```diff
@@ -30,6 +30,7 @@ interface VariantManager {
     // variant-dependant features listed here
     sealed class VariantFeature {
         object StickySearchNotification : VariantFeature()
+        object BottomBarNavigationExperiment : VariantFeature()
```

Comment:
@malmstein I know you've closed this but a quick naming suggestion from me, let's omit `experiment` and call this `BottomBarNavigation`. Features shouldn't have knowledge of experiments - experiments list their features. It's important to keep the concepts clear. When we have multiple experiments running we sometimes reuse features combinations across them and it can get confusing.

Likewise anywhere in code where you check for this feature make sure the wording is feature based e.g `inflateBottomBarWithSearchFeature` rather than `inflateBottomBarWithSearchExperiment`.